openpype/modules/extra_addons/process_monitor/addon.py

```diff
@@ -0,0 +1,704 @@
+"""
+
+psutil 'process_iter' parameters:
+'cmdline', 'connections', 'cpu_affinity', 'cpu_num', 'cpu_percent',
+'cpu_times', 'create_time', 'cwd', 'environ', 'exe', 'gids', 'io_counters',
+'ionice', 'memory_full_info', 'memory_info', 'memory_maps', 'memory_percent',
+'name', 'nice', 'num_ctx_switches', 'num_fds', 'num_handles', 'num_threads',
+'open_files', 'pid', 'ppid', 'status', 'terminal', 'threads', 'uids', 'username'
+
+"""
+
+import os
+import click
+
+import threading
+import ctypes
+import platform
+
+from openpype.lib import Logger
+
+#TODO: Test with Python2
+import psutil
+
+
+from openpype.modules import (
+    JsonFilesSettingsDef,
+    OpenPypeAddOn,
+    ModulesManager,
+    IPluginPaths,
+    ITrayAction
+)
+
+from openpype.api import get_anatomy_settings, get_system_settings
```

Comment:
Please don't use `openpype.api` imports anymore.

Reply:
To be sure,
I should import ```openpype.settings``` for these 2 functions.

What about the ```Logger```?
Should I also change and import ```openpype.lib``` instead?